test/blackbox-tests/test-cases/inline_tests/parallel.t/run.t

```diff
@@ -5,14 +5,14 @@ First, build silently to avoid some noise
 See that `test1/runtest`, which uses `fake_backend_1, only runs one inline test runner
 
   $ dune build --display short @test1/runtest 2>&1 | grep alias
-  inline_test_runner_test_lib1 alias test1/runtest
+           run alias test1/runtest
 
 See that `test2/runtest`, which uses `fake_backend_2`, runs one inline test runner per partition
 
   $ dune build --display short @test2/runtest 2>&1 | grep alias
-  inline_test_runner_test_lib2 alias test2/runtest
-  inline_test_runner_test_lib2 alias test2/runtest
-  inline_test_runner_test_lib2 alias test2/runtest
+           run alias test2/runtest
+           run alias test2/runtest
+           run alias test2/runtest
```

Comment:
One could argue that the new output is less informative than the old one, but I don't see how we can do better while keeping the paths short.

Reply:
I've pushed alternative names. the inline-test executable in now called `inline-test-runner`. Its entry point is `main.ml`

Reply:
Thanks. I'm a bit hesitatnt to merge because I have never used inline tests myself so I am not 100% sure I am not missing any details. A quick glance from someone familiar with this part of the code (or @rgrinberg) would be good.